src/collection/components/CollectionFragment.tsx

```diff
@@ -243,6 +243,7 @@ const CollectionFragment: FunctionComponent<CollectionFragmentProps> = ({
 			resultPath="app_item_meta[0]"
 			renderData={renderCollectionFragment}
 			notFoundMessage="De meta item van deze collectie werd niet gevonden"
+			showSpinner={false}
```

Comment:
Ga je zo niet gwn geen enkele spinner meer tonen op de pagina?

Reply:
In deze component zetten we hem op false maar de collection fetch erboven op de bovenstaande pagina heeft nog altijd zijn spinner.